docs/client/session.md

```diff
@@ -0,0 +1,25 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
+
+The primary purpose of sessions is to provide insight into the series of user actions or events that lead a critical or 
+interesting occurrence in the application's use. Sessions also provide a means to quantify overall application usage.
+
+
+## Attributes
+
+This attribute can be applied to all signals generated by an agent. It may be omitted from metrics, due to their aggregated nature. 
```

Comment:
We could instead say: Client apps generally do not emit metrics since they represent only one user, but in case metrics are emitted then the following attribute should be applied to it as well.

Reply:
This brings up a good point -- sessions IDs would always be a high-cardinality dimension on a metric. Something I hadn't considered before. Scope level attributes are an interesting possibility, but perhaps components (meter, tracer) have a lifespan that could outlive a session...which poses a different challenge.

Reply:
> Client apps generally do not emit metrics since they represent only one user 

I actually don't think that's true. They may be interpreted in a different way, but it's a valid use case to associate metrics (in the generic sense) to both sessions and users.

> (meter, tracer) have a lifespan that could outlive a session

Spans can also out live a session depending on how you define a session and a span. The relationship is a fun many-to-many one :-)

Reply:
I agree, with @breedx-splk , `session.id` is a high cardinality field. I don't think it is a good idea to ever have that as metric attributes.